instrumentation/jmx-metrics/library/src/test/java/io/opentelemetry/instrumentation/jmx/rules/assertions/JmxAssertj.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jmx.rules.assertions;
+
+import io.opentelemetry.proto.metrics.v1.Metric;
+
+/** Dedicated Assertj extension to provide convenient fluent API for metrics testing */
+// TODO: we should contribute this back to sdk-testing
+// This has been intentionally not named `*Assertions` to prevent checkstyle rule to be triggered
+public class JmxAssertj extends org.assertj.core.api.Assertions {
```

Comment:
[for reviewer] this has been copied as-is from contrib with a small rename https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/assertions/Assertions.java, it would make sense to add those assertions to `io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions` but the latter is in the SDK, works with `MetricData` and not directly with the `io.opentelemetry.proto.metrics.v1.Metric` class.